tm2/pkg/crypto/keys/client/addpkg.go

```diff
@@ -1,222 +0,0 @@
-package client
```

Comment:
The file is marked as deleted, but it was actually moved and then modified. We want to preserve the file's history. To handle the rename and edit, we can make two commits or explore other options. cc @harry-hov.

Reply:
hmm, from an history point view, it's correctly mark as renamed in the commit: 32511b39c51fdecf2e5b10f89cae5323a5ca1e5
but since this PR will be squash when merge, im not sure we can do anything about it...